### ISSUE TITLE
ghpages only reading README.md

### ISSUE THREAD

**Issue by hjpretty** [hjpretty/authorities#2]
Can't get the ghpages to recognize any other file but the README.md

**Comment by hjpretty** [comment]
The current "fix" is to put my content into the "README.md" file.
This is a crude fix, but I don't have time to figure out a better one right now.

**Comment by chrisdaaz** [comment]
It's probably not worth your time because I like the site you currently have and it's not obvious to me that you need more than that. If you wanted to figure out how to expand your site, you would need to become familiar with [Jekyll](https://jekyllrb.com/docs/), the website building tool you're currently using to convert the README.md to a website. There are good tutorials you can find on YouTube that walk through everything step by step. It can get complicated quickly, though.

**Comment by hjpretty** [comment]
Agreed. In my project workflow, this issue is placed in priority after "learning more about Git and GitHub". I figure if as I'm learning more about Git and GitHub I come across a solution for this, that's great. Otherwise, this is functional and works fine as is.
